src/main/resources/openapiv2.json

```diff
@@ -652,6 +686,103 @@
                     "type": "string"
                 }
             ]
+        },
+        "/consumers/{groupid}/instances/{name}/records": {
+            "get": {
+                "tags": [
+                    "Consumers"
+                ],
+                "description": "Retrieves records for subscribed consumer, including message values, topics, and partitions.",
```

Comment:
even in this case we have to hightlight that the request must be sent to the Kafka bridge instance where the consumer instance lives.

Reply:
Retrieves records for a subscribed consumer from the Kafka Bridge instance that holds it, including message values, topics, and partitions.